sdk/sdk.yaml

```diff
@@ -227,120 +227,120 @@ paths:
               schema:
                 $ref: sdk_responses.yaml#/alchemy_getAssetTransfers
   /{apiKey}/getBalance:
-      post:
-        operationId: sdk-getBalance
-        summary: getBalance
-        description: Returns the balance of the account of a given address.
-        tags: ['SDK Core Endpoints']
-        parameters:
-          - name: apiKey
-            in: path
+    post:
+      operationId: sdk-getBalance
+      summary: getBalance
+      description: Returns the balance of the account of a given address.
+      tags: ['SDK Core Endpoints']
+      parameters:
+        - name: apiKey
+          in: path
+          schema:
+            type: string
+            default: docs-demo
+            description: For higher throughput, **[create your own API key](https://alchemy.com/?a=docs-demo)**
+          required: true
+      requestBody:
+        content:
+          application/json:
             schema:
-              type: string
-              default: docs-demo
-              description: For higher throughput, **[create your own API key](https://alchemy.com/?a=docs-demo)**
-            required: true
-        requestBody:
+              $ref: sdk_body.yaml#/eth_getBalance
+      x-readme:
+        explorer-enabled: false
+        samples-languages:
+          - javascript
+        code-samples:
+          - language: javascript
+            name: Alchemy SDK
+            code: |
+              // Setup: npm install alchemy-sdk
+              // Github: https://github.com/alchemyplatform/alchemy-sdk-js
+              import { Network, Alchemy } from "alchemy-sdk";
+
+              // Optional config object, but defaults to demo api-key and eth-mainnet.
+              const settings = {
+                apiKey: "demo", // Replace with your Alchemy API Key.
+                network: Network.ETH_MAINNET, // Replace with your network.
+              };
+              const alchemy = new Alchemy(settings);
+
+              alchemy.core.getBalance("vitalik.eth", "latest").then(console.log);
```

Comment:
@SahilAujla this sample code won't actually run / compile sadly - this is why we've typically "hidden" the try it button on all sdk api references